### PR TITLE
Apidoc link function

### LINKED ISSUE #3369: URLs for specific API commands
Commit id:       f7029eddd4120f672c59a4b3a4395832f4de8863

When visiting http://api-docs.cuberite.org, I'd want to have links to individual commands via the hash part of the URL. e.g. `http://api-docs.cuberite.org/cPlayer.html#Freeze` Should lead to the player class's `Freeze` command.

This should be rather simple: Each command's first HTML table cell, `<td>X</td>` should be changed to:

```
<td id="X">
  <a href="PageURL#X">X</a>
</td>
```

Better methods are welcome.
This may require changes to the built in APIDump plugin, which is why I posted it here.

### PR DESCRIPTION
**Pull request by madmaxoft**
Fixes #3369 .

### CODE REVIEW

File: Server/Plugins/APIDump/main_APIDump.lua [hunk 1]
```diff
@@ -778,12 +778,19 @@ local function WriteHtmlClass(a_ClassAPI, a_ClassMenu)
...
 			local StaticClause = ""
 			if (func.IsStatic) then
 				StaticClause = "(STATIC) "
 			end
-			cf:write("<tr><td>", func.Name, "</td>\n");
+
+			-- Increase number by one
+			TableOverloadedFunctions[func.Name] = (TableOverloadedFunctions[func.Name] or 0) + 1
+			-- Add the anchor names as a title
+			cf:write("<tr><td id=\"" .. func.Name .. "_" .. TableOverloadedFunctions[func.Name] .. "\" title=\"" .. func.Name .. "_" .. TableOverloadedFunctions[func.Name] .."\">", func.Name, "</td>\n");
```
Comment (madmaxoft) [inline Server/Plugins/APIDump/main_APIDump.lua]: Use comma instead of concatenation: `cf:write(string1, string2, string3)`

**Comment by madmaxoft** [comment]
Replace the concatenations with commas, squash into a single commit and enter a shorter commit message, and all done.

**Comment by Seadragon91** [comment]
@madmaxoft Squashed and replaced `..` with comma.

@LogicParrot Is that okay for you?
Examples:
cPlayer.html#GetClientHandle_1
cPlayer.html#MAX_FOOD_LEVEL 
cChestEntity.html#GetContents_1

**Comment by madmaxoft** [comment]
Looks good to me.

**Pull request merged**